src/ebay-listbox-button/listbox-button.tsx

```diff
@@ -202,13 +214,18 @@ const ListboxButton: FC<EbayListboxButtonProps> = ({
     })
     const expandBtnTextId = prefixId && 'expand-btn-text'
 
+    useEffect(() => {
```

Comment:
This is dangerous, we need to do it where we do setExpanded instead. The reason is that the first render it will trigger onCollapse even if the user didn't make any action. In case someone is tracking onCollapse it will trigger tracking without being necessary